src/CoreBundle/Resources/public/javascript/jquery/jQueryUI/themes/cupertino/cupertino.css

```diff
@@ -147,10 +147,7 @@
 .ui-widget .ui-widget {
     font-size: 11px;
 }
-.ui-widget input, .ui-widget select, .ui-widget textarea, .ui-widget button {
-    font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;
-    font-size: inherit;
-}
+
```

Comment:
This is a jQueryUI file - does it make sense to change this file? Both as it's third-party and as we plan to remove jQueryUI completely. Or is it meant as an intermediary step anyway?

Reply:
It is intermediate. We need this change as long as we have jquery ui in the core, but this would overwrite the bootstrap 4 styles.